lib/datadog/tracing/metadata/ext.rb

```diff
@@ -1,4 +1,6 @@
 # typed: true
+# frozen_string_literal: true
```

Comment:
I add this magic comment to ensure strings I introduced here:
```ruby
def to_tag(name)
  tag = name.to_s.strip
  tag.downcase!
  tag.gsub!(INVALID_TAG_CHARACTERS, '_')
  tag.tr!('.', '_')
  tag
end
```
are not recreated on every invocation.

The downside is that Rubocop does not let me keep all the `.freeze` statements in this file, so I had to change those :/

Reply:
Sometimes Rubocop leaves much to be desired.